code-challenges.js

```diff
@@ -58,12 +110,27 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+const indexOfValue = (value) => {
+    return numberOfConnections.lastIndexOf(value)
+}
+console.log(indexOfValue(givenValue1))
```

Comment:
How can you refactor without hard coding the `numberOfConnections` variable in the function? Also, in your pseudo code, you wrote that the output is the last index of array, which is not quite what the expected output should be.